packages/expo-doctor/src/checks/__tests__/GlobalPrereqsVersionCheck.test.ts

```diff
@@ -0,0 +1,60 @@
+import spawnAsync from '@expo/spawn-async';
+import { execSync } from 'child_process';
+
+import { asMock } from '../../__tests__/asMock';
+import { GlobalPrereqsVersionCheck } from '../GlobalPrereqsVersionCheck';
+
+jest.mock('child_process');
+
+// required by runAsync
+const additionalProjectProps = {
+  exp: {
+    name: 'name',
+    slug: 'slug',
+  },
+  pkg: {},
+};
+
+describe(GlobalPrereqsVersionCheck, () => {
```

Comment:
we generally avoid redundant describe blocks (single blocks which match the name of the test file).